<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionVerbal</class>
 <widget class="QWidget" name="QuestionVerbal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verbal Ability</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/img/Img/logo.jpg</normaloff>:/new/img/Img/logo.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(242, 255, 181);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QPushButton" name="pushButton_Submitans">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>590</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20%;
background-color: rgb(255, 200, 152);
color:red;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qno">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Q. NO)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>1161</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Question</string>
   </property>
   <widget class="QRadioButton" name="rop1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rop2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rop3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rop4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_question">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Question</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Verbal Ability</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cname">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>COMPANY NAME </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_right">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
border:2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_wrong">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>20</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
border:2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Wrong</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
